components/omega/test/CMakeLists.txt

```diff
@@ -405,3 +405,18 @@ add_omega_test(
     ocn/TracersTest.cpp
     "-n;8"
 )
+
+##################
+# EOS tests
+##################
+
+add_test(NAME GSWC_CALL_TEST
+    COMMAND ${OMEGA_BUILD_DIR}/external/GSW-C/gsw_check
+)
+
+add_omega_test(
+    EOS_TEST
+    testEos.exe
+    ocn/EosTest.cpp
+    "-n;1"
```

Comment:
Might consider using more than one task for the test just to make sure configurations  and other stuff are performed correctly.

Reply:
Changed to 2.